src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -219,13 +221,14 @@ private void prepareRequestForAttachingDocumentToCase(CaseDocumentsMetadata case
                     caseDocumentsMetadata.getJurisdictionId());
                 metadataMap.put(Constants.JURISDICTION_ID, caseDocumentsMetadata.getJurisdictionId());
             }
-
             DocumentUpdate documentUpdate = new DocumentUpdate();
             documentUpdate.setDocumentId(UUID.fromString(documentHashToken.getId()));
-
             documentUpdate.setMetadata(metadataMap);
-            bodyMap.add("documents", documentUpdate);
+
+            list.add(documentUpdate);
         }
+
+        return new UpdateDocumentsCommand(null, list);
```

Comment:
Could we have a self describing constant for the null value to avoid having to work out what expected parameters are for `UpdateDocumentsCommand` constructor

`private static final Date NULL_TTL = null`

and then

`return new UpdateDocumentsCommand(NULL_TTL, list);`